Documentation/commands.md

```diff
@@ -209,8 +209,37 @@ FOO=bar
 EXAMPLE_OVERRIDE=over
 ```
 
+#### Logging
+
+To get the logs you need to get pod's machine name. You can use machinectl
+
+```
+$ machinectl
+MACHINE                                  CLASS     SERVICE
+rkt-f241c969-1710-445a-8129-d3a7ffdd9a60 container nspawn
+
+1 machines listed.
+```
+
+or `rkt list --full`
+
+```
+# rkt list --full
+UUID                                    ACI     STATE   NETWORKS
+f241c969-1710-445a-8129-d3a7ffdd9a60    busybox running
+```
+
+Pod's machine name will be the pod's UUID with a `rkt-` prefix.
+
+Then you can use systemd's journalctl with `_HOSTNAME` set to the machine name:
+
+```
+$ journalctl -m _HOSTNAME=rkt-f241c969-1710-445a-8129-d3a7ffdd9a60
```

Comment:
`journalctl -M` will not be fixed in this PR and will be dealt with later (#947). I suggest to add the following in the documentation:

`rkt` does not integrate with `journalctl -M` yet (see #947).